kubernetes/README.md

```diff
@@ -3,35 +3,36 @@
 ## Overview
 
 TorchElastic Controller for Kubernetes manages a Kubernetes custom resource `ElasticJob` and makes it easy to
-run Torch Elastic workloads on Kubernetes.   
+run Torch Elastic workloads on Kubernetes.
 
 ### Prerequisites
 
 - Kubernetes >= 1.12
 - [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl)
 - [kustomize](https://github.com/kubernetes-sigs/kustomize/blob/master/docs/INSTALL.md)
 
-> **NOTE**: 
+> **NOTE**:
 >
 >  1. (recommended) create a cluster with GPU instances as some examples
 >     (e.g. imagenet) only work on GPU.
 >  2. If you provision instances with a single GPU you will only be able to run
 >     a single worker per node.
->  3. Our examples assume 1 GPU per node so you will have to adjust 
+>  3. Our examples assume 1 GPU per node so you will have to adjust
 >     `--nproc_per_node` to be equal to the number of CUDA devices
->     on the instance you are using if you want to run multiple workers per 
+>     on the instance you are using if you want to run multiple workers per
 >     container
 
 ### (Optional) Setup
 
-Here we provide the instructions to create an Amazon EKS cluster. If you 
-are not using AWS please refer to your cloud/infrastructure provider's manual
-to setup a kubernetes cluster. 
+Here we provide the instructions to create an Amazon EKS cluster or a Microsoft AKS cluster. If you
+are not using AWS or Azure, please refer to your cloud/infrastructure provider's manual
+to setup a kubernetes cluster.
 
-> **NOTE**: EKS is not required to run this controller, 
+> **NOTE**: EKS/AKS is not required to run this controller,
 >  you can use other Kubernetes clusters.
 
-Use `eksctl` to create an Amazon EKS cluster. This process takes ~15 minutes. 
+#### Create your cluster
+1. Use `eksctl` to create an Amazon EKS cluster. This process takes ~15 minutes.
```

Comment:
nit: how about something like this? 
`1. 2. `feels a little confusing here.

```
### (Optional) Setup
#### Create your cluster
##### Create EKS
##### Create AKS
#### Install Nvidia device plugin to enable GPU support on your cluster.
```

Reply:
updated